include/string_utils.h

```diff
@@ -311,18 +311,31 @@ enum class UnicodeFallback {
 	Box
 };
 
+// Specifies how to interpret characters 0x00-0x1f and 0x7f in DOS strings
+enum class DosStringType {
```

Comment:
Technically, it's a bit incorrect to call it like this. You're controlling the conversion logic, the DOS string is a DOS string, just a bunch of bytes. Maybe think of a better name? `DosStringConvertMode`?

Reply:
OK, changed.